File: Exercism Codes/Java/Proverb.java
class Proverb {
    public String[] words;

    Proverb(String[] words) {
        this.words = words;
    }

    String recite() {
        String sentence = "";
        for(int i = 0; i < words.length; i++){
            if(i < words.length - 1){
            sentence += "For want of a " + words[i] + " the " + words[i + 1] + " was lost.\n";
            } else {
                sentence += "And all for the want of a " + words[0] + ".";
            }
        }
        return sentence;
    }

}

File: Exercism Codes/C#/Proverb.cs
public static class Proverb
{
    public static string[] Recite(string[] subjects)
    {
        List<string> sentence = new List<string>();
        for (int i = 0; i < subjects.Length; i++)
        {
            if (i < subjects.Length - 1)
            {
                sentence.Add($"For want of a {subjects[i]} the {subjects[i + 1]} was lost.");
            }
            else
            {
                sentence.Add($"And all for the want of a {subjects[0]}.");
            }
        }
        return sentence.ToArray();
    }
}